modules/monitoring-certificates.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-certificates.adoc
+
+[id="monitoring-certificates_{context}"]
+= Monitoring certificates
+
+
+TBD:
+
```

Comment:
@deads2k Can you please help me build out this section? The customer is asking for the information outlined here.

Reply:
@deads2k Can you please help provide information by end of week? The information we are needing for monitoring certificates includes: 
* The purpose
* File path
* Default expiration term
* How to set custom expiration term
* How to specify the expiration date of all certificates used in OpenShift when installing OpenShift
* Which services use it
* How to update/extend it
* Will certificates that are about to expire be automatically renewed by the operator?

Reply:
I don't know anything about any monitoring certificates.  Perhaps @s-urbaniak ?

Reply:
There is some amount of client cert monitoring done by the kube api and we have alerts on this: https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/a7ee9d1abe1b1a3670a02ede1135cadb660b9d0c/alerts/kube_apiserver.libsonnet#L125-L148

I don't believe there is any serving certs monitoring, as this is typically done via blackbox probes which is not something the monitoring team has available as part of the monitoring framework today (in the same way as teams can self serve scraping/alerting with the Prometheus Operator CRDs).

Reply:
@brancz how do we document that? Is there a way to pull/show what Prometheus Operator CRDs rules the individual operators are providing?

Reply:
@brancz Can you please comment?